sei-cosmos/x/capability/module.go

```diff
@@ -155,11 +153,3 @@ func (am AppModule) ExportGenesisStream(ctx sdk.Context, cdc codec.JSONCodec) <-
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
 func (AppModule) ConsensusVersion() uint64 { return 1 }
-
-// BeginBlocker calls InitMemStore to assert that the memory store is initialized.
-// It's safe to run multiple times.
-func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
-	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
-
-	am.keeper.InitMemStore(ctx)
```

Comment:
why was this not previously calling capability's `BeginBlocker` function which also just calls the `InitMemStore`?